etc/skydive.yml.default

```diff
@@ -47,6 +47,10 @@ analyzer:
   # topology_filter:
   #   Namespaces: "g.V().Has('Type', 'netns').OutE().BothV()"
   #   Layer2: "g.E().Has('RelationType', 'layer2')"
+  # By default no capturing,  set filter to capture from selected nodes
+  # from the beginning automatically
+  # startup_capture: "G.V().has('Name', NE('lo'))"
+  # startup_capture_bpf: "port 80"
```

Comment:
I suggest grouping captures (as I expect configuration for startup_capture will grow) and adding a specific flag to activate or enable.

```
analyzer:
  capture:
    gremlin_filter: "G.V().has('Name', NE('lo'))"
    bpf_filter: "port 80"
    automatic: true
```

Reply:
I agree with you, good idea!
Thanks!

Reply:
changed it according in a bit different way.